lib/std/crypto/25519/field25519.zig

```diff
@@ -0,0 +1,327 @@
+const std = @import("std");
```

Comment:
Could this module be renamed to simply `field.zig` or do we expect more field types under `25519`?

Reply:
This module can be renamed, but since all crypto primitives are directly exported in the `crypto` namespace how to avoid collisions at that level?

Reply:
Hmm, would there be any collisions though? I thought we don't re-export `field25519` and `scalar25519` or am I wrong here?

Reply:
Since `field` and `scalar` are re-exported by the curve structures, looks like the answer is to not export them in `crypto` :)

Reply:
Oh, and actually, they're not :) Let me rename these modules then!